src/screens/Profile/Profile.js

```diff
@@ -119,7 +119,11 @@ const emailFormFields = [{
   label: 'Email',
   name: 'email',
   type: 'email',
-  config: { placeholder: 'user@example.com', autoCapitalize: 'none', error: 'Please specify valid email' },
+  config: {
```

Comment:
I'm not sure where do we use this view, I just changed it to make sure the functions I was changing were up-to-date everywhere they were being used. While I was here, I cut this line because it was too long, did I break it?

Reply:
Nope, I've just noticed that it does not work as intended. Your changes had no influence on it. We can address it some other time :)